AzFilesHybrid/AzFilesHybrid.psm1

```diff
@@ -3755,10 +3755,16 @@ function Debug-AzStorageAccountAuth {
         
                 if (!(Get-IsDomainJoined))
                 {
-                    $message = "Machine is not domain-joined. Mounting to Azure Files through" `
-                        + " Active Directory Authentication is only supported when the computer is joined to" `
-                        + " an Active Directory domain, which is synced to Azure AD with Azure AD Connect" `
-                        + " (https://docs.microsoft.com/en-us/azure/storage/files/storage-files-identity-auth-active-directory-enable#prerequisites)"
+                    $message = "Machine is not domain-joined." `
+                        + "Machine is not domain-joined.\n\n" `
+ 
+                        + "Being domain-joined to an AD DS domain is a prerequisite for mounting" `
+                        + " Azure file shares without having to explicitly provide user credentials at every mount." `
+                        + " (https://docs.microsoft.com/en-us/azure/storage/files/storage-files-identity-auth-active-directory-enable#prerequisites).\n\n" `

```

Comment:
Let's edit the links to be consistent between here and the one below.

I would suggest not skipping a line before the link, but instead just appending the sentence "See [link]" to the end of the paragraph.